src/extras/core/Curve.js

```diff
@@ -110,9 +106,7 @@ Object.assign( Curve.prototype, {
 
 	// Get list of cumulative segment lengths
 
-	getLengths: function ( divisions ) {
-
-		if ( divisions === undefined ) divisions = this.arcLengthDivisions;
+	getLengths: function ( divisions = this.arcLengthDivisions ) {
```

Comment:
Can you actually use `this` at this point? Have you tested this code?

Reply:
I saw the `build/three.js` it was work, TBH, it looks odd.